packages/params/src/forkName.ts

```diff
@@ -96,6 +96,28 @@ export function isForkPostElectra(fork: ForkName): fork is ForkPostElectra {
   return isForkPostDeneb(fork) && fork !== ForkName.deneb;
 }
 
+export type ForkPreFulu = ForkPreElectra | ForkName.electra;
+export type ForkPostFulu = Exclude<ForkName, ForkPreFulu>;
+export const forkPostFulu = exclude(forkAll, [
+  ForkName.phase0,
+  ForkName.altair,
+  ForkName.bellatrix,
+  ForkName.capella,
+  ForkName.deneb,
+  ForkName.electra,
+]);
+export function isForkPostFulu(fork: ForkName): fork is ForkPostFulu {
+  return isForkPostElectra(fork) && fork !== ForkName.electra;
+}
+
+export type ForkPreBlobs = ForkPreDeneb;
```

Comment:
can we get rid of this, or we do we still use `ForkBlobs` instead of `ForkPostDeneb`?

Reply:
> But forkBlobs now means "forks in which blobs are the primary DA mechanism" so has a unique meaning.

we need to get rid of this notion of using features to determine forks, it's confusing

also noticed `export type ForkPostBlobs = ForkPostFulu;` and then `forkBlobs = [ForkName.deneb, ForkName.electra];` lol

Reply:
why does fulu not have blobs? it still has blobs in many places, just not on p2p

Reply:
> lots of places with isForkPostDeneb && isForkPreFulu which stinks

seems architectural issue in the code, having a fork guard that just covers a range of forks is horrible design and very confusing, and doesn't the EL literally return us blobs? so claiming Fulu has no blobs is just technically incorrect as well

> isForkPostDeneb && isForkPreFulu

that's a lot better than `isForkBlobs` at least we didn't make up some new term that only the ones that invented it can understand

Reply:
or something like `isForkBetween()` or similar, anything more explicit I would be fine with